gobblin-iceberg/src/main/java/org/apache/gobblin/iceberg/Utils/IcebergUtils.java

```diff
@@ -248,6 +248,7 @@ public static DataFile getIcebergDataFileWithMetric(org.apache.gobblin.metadata.
         IcebergUtils.getMapFromIntegerLongPairs(file.getFileMetrics().getColumnSizes(), schemaIdMap),
         IcebergUtils.getMapFromIntegerLongPairs(file.getFileMetrics().getValueCounts(), schemaIdMap),
         IcebergUtils.getMapFromIntegerLongPairs(file.getFileMetrics().getNullValueCounts(), schemaIdMap),
+        IcebergUtils.getMapFromIntegerLongPairs(Lists.newArrayList(), schemaIdMap), // metric value will be null since Nan values are supported from avro version 1.10.*
```

Comment:
Do we know which metric is added? I don't believe we should give Nan here. It's true in avro version, it's null but how about ORC files? https://github.com/apache/gobblin/pull/3697/files#diff-8ed59fcb5438eea6df2d8ed2256d2eac003a523c7af29aeef7fd6a7a151fbbabR214 we do get all metrics for ORC files, so we should publish it here as well.

Reply:
I have added a TODO comment in case we need to handle the file metric conversion for ORC in my latest commit!